### PR TITLE
[WIP] List and fulfil booking requests

### PR DESCRIPTION
**Pull request by benbarnett**
Introduces `govuk_admin_template` gem for shared styles and lists the booking requests.

### CODE REVIEW

File: app/views/booking_requests/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Hackney</h1>
```
Comment (dwhenry) [inline app/views/booking_requests/index.html.erb]: I thought we had more than one booking location for the pilot?

Comment (benbarnett) [inline app/views/booking_requests/index.html.erb]: Yep none of this is plugged into the locations API yet

**Comment by dwhenry** [comment]
remove the helper file and then 🚢

**Comment by benlovell** [comment]
Sweet. Can we leave this sitting around though, so it can be picked up in the context of a story?

**Comment by benbarnett** [comment]
Yeah sorry this was meant to be more a WIP thing anyway

**Comment by benlovell** [comment]
Closing this as I've cherry-picked what I needed in the `fulfilment` branch that's near to a PR.

**Pull request closed without merging**